app/src/main/AndroidManifest.xml

```diff
@@ -123,13 +147,17 @@
         <service
             android:name=".services.TimerService"
             android:foregroundServiceType="specialUse">
-            <property android:name="android.app.PROPERTY_SPECIAL_USE_FGS_SUBTYPE" android:value="Used to notify the user that timers are running"/>
+            <property
+                android:name="android.app.PROPERTY_SPECIAL_USE_FGS_SUBTYPE"
+                android:value="Used to notify the user that timers are running" />
```

Comment:
where is this value shown, to the user?

Reply:
According to https://developer.android.com/about/versions/14/changes/fgs-types-required :
"These values and corresponding use cases are reviewed when you submit your app in the Google Play Console."

So I think these are not really meant for users, but for Play Store review.